relay-general/src/store/normalize.rs

```diff
@@ -34,6 +37,18 @@ mod stacktrace;
 
 mod user_agent;
 
+/// Configuration for measurements normalization.
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase")]
+pub struct MeasurementsConfig {
+    /// A list of measurements that are built-in and are not subject to custom measurement limits.
+    #[serde(default, skip_serializing_if = "BTreeSet::<String>::is_empty")]
+    known_measurements: BTreeSet<String>,
```

Comment:
Same suggestion from sentry's PR: I'd rename this to something similar to `builtin_measurements` in order to have one name per concept (and we use "built-in" quite extensively in general).